.circleci/config.yml

```diff
@@ -15,6 +15,10 @@ executors:
     docker:
       - image: cirrusci/flutter
     shell: "/bin/bash -eo pipefail"
+  node-executor:
```

Comment:
Could probably remove the orb if we are going to go this route I think? Figured it was either this approach or changing 

```    executor:
      name: node/default
      tag: "14"
```
      
      to 
      
```    executor:
      name: node/default
      tag: "14.20"
```